be/src/exec/base_scanner.cpp

```diff
@@ -144,12 +145,21 @@ bool BaseScanner::fill_dest_tuple(const Slice& line, Tuple* dest_tuple, MemPool*
         ExprContext* ctx = _dest_expr_ctx[dest_index];
         void* value = ctx->get_value(_src_tuple_row);
         if (value == nullptr) {
-            if (_strict_mode && (_src_slot_descs_order_by_dest[dest_index] != nullptr)
-                && !_src_tuple->is_null(_src_slot_descs_order_by_dest[dest_index]->null_indicator_offset())) {
+            SlotDescriptor* slotDescriptor = _src_slot_descs_order_by_dest[dest_index];
+            if (_strict_mode && (slotDescriptor != nullptr)&& !_src_tuple->is_null(slotDescriptor->null_indicator_offset())) {
+                StringValue* rawValue = _src_tuple->get_string_slot(slotDescriptor->tuple_offset());
                 std::stringstream error_msg;
-                error_msg << "column(" << slot_desc->col_name() << ") value is incorrect "
-                    << "while strict mode is " << std::boolalpha << _strict_mode;
-                _state->append_error_msg_to_file("", error_msg.str());
+                if (rawValue) {//is not null then get raw value
+                    std::string rawString = rawValue->to_string();
+                    error_msg << " column(" << slot_desc->col_name() << ") value("<< rawString <<") is incorrect "
+                        << "while strict mode is " << std::boolalpha << _strict_mode;
+                    _state->append_error_msg_to_file(rawString, error_msg.str());
+                } else {
+                    error_msg << " column(" << slot_desc->col_name() << ") value is incorrect "
+                        << "while strict mode is " << std::boolalpha << _strict_mode;
+                    _state->append_error_msg_to_file("", error_msg.str());
+                }
+                LOG(WARNING) << error_msg.str();
```

Comment:
remove this log, we should not print error log for wrong data when load. This is a normal case.

Reply:
ok